BuyPolish/Pola.xcodeproj/project.pbxproj

```diff
@@ -1681,9 +1689,9 @@
 				GCC_TREAT_INCOMPATIBLE_POINTER_TYPE_WARNINGS_AS_ERRORS = YES;
 				GCC_TREAT_WARNINGS_AS_ERRORS = YES;
 				INFOPLIST_FILE = "Pola/SupportingFiles/Pola-Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
Jedyne zmiany w project.pbxproj jakie powinny być to te z dodaniem pliku krasnystaw.json.